### PR TITLE
improvement: created a reusable button component Issue #20

### PR DESCRIPTION
**Pull request by gagan-aryan**

### CODE REVIEW

File: client/src/pages/chatlink/components/button.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import '../src/pages/chatlink/style.css';
+
+const Button = (props) => {
```
Comment (muke1908) [inline client/src/pages/chatlink/components/button.js:4]: Let's destructure the props here, will be more readable and look clean.
`const { type, state, onClick, width } = props`

File: client/src/pages/chatlink/components/button.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  
+  const { label, type, state, onClick, width } = props
+  
+  const styles = {
+    background: type == 'secondary' ? '#4caf50' : '#3944bc',
+    color: '#fff',
+    padding: '10px',
+    width: width,
```
Comment (muke1908) [inline client/src/pages/chatlink/components/button.js:11]: `width` should be optional. Often time we would want the button to take the width automatically based on the content.

Suggestion:
```
width: width || 'auto',
display: inline-block
```
`alignItem`, `justifyContent` - not really required in this case..

**Comment by gagan-aryan** [comment]
Sorry for way too many redundant commits. Won't happen next time.

**Comment by muke1908** [comment]
> Sorry for way too many redundant commits. Won't happen next time.

There is `sqaush` for that in git :D  No worry!

**Pull request merged**